src/Chainweb/Pact/PactService.hs

```diff
@@ -466,10 +470,11 @@ applyPactCmd isGenesis (Env' dbEnv) cmdIn miner = do
 
     -- cvt from Command PayloadWithTexts to Command ((Payload PublicMeta ParsedCode)
     let !cmd = payloadObj <$> cmdIn
-    result <- liftIO $ if isGenesis
+    T2 result mcache' <- liftIO $ if isGenesis
         then applyGenesisCmd logger dbEnv pd spv cmd
-        else applyCmd logger dbEnv miner gasModel pd spv cmd
-    pure $! toHashCommandResult result
+        else applyCmd logger dbEnv miner gasModel pd spv cmd mcache
+
+    pure $! T2 (V.cons (toHashCommandResult result) v) mcache'
```

Comment:
How "hot path" is this `V.cons`? Vector consing can get expensive.

Reply:
We're only doing <blocksize> many of these a per block. Perhaps I could make these a list and `fromList` the result

Reply:
Where blocksize == TX count? That could be 1000.

Reply:
Yes